slick.grid.js

```diff
@@ -2116,6 +2117,11 @@ if (typeof Slick === "undefined") {
       var i;
       if (!stylesheet) {
         var sheets = (options.shadowRoot || document).styleSheets;
+
+        if (options.devMode?.ownerNodeIndex >= 0) {
```

Comment:
so just to be clear, the `devMode` can be of type `boolean` or `{ ownerNodeIndex?: number; containerClientWidth?: number; }`, is that correct? I'm asking because I'll have to backport this to the SlickGrid v5 and also Slickgrid-Universal upcoming major version

Reply:
exactly right. I can also do the backport to v5(master?!) as mentioned above.

Reply:
alright I'm ok with the PR then as mentioned in paragraph below. So let me go ahead and merge the PR, I'll let you do the backport to v5, thanks for that. 

I'll push new SlickGrid versions and other upstream libs probably in the weekend.
On all my upstream libs, I've now switched to GitHub Action workflows to publish releases because I wanted to publish on npm with `--provenance` and it's now much quicker to push releases, just hit start workflow :)

Reply:
sounds great. getting late over here so I'll open the PR for v5 tomorrow. thx